view_all_inc.php

```diff
@@ -111,10 +111,15 @@
 		<div class="btn-toolbar">
 			<div class="btn-group pull-left">
 		<?php
-			# -- Print and Export links --
+			# -- Print link --
```

Comment:
I don"t know which tool has your preference to log the discussion. Here or in issue : 
http://www.mantisbt.org/bugs/view.php?id=22224#c55574